.github/workflows/testing.yml

```diff
@@ -74,7 +74,39 @@ jobs:
       - name: Test
         env:
           MP_API_KEY: ${{ secrets.MP_API_KEY }}
-        run: pytest --cov=atomate2 --cov-report=xml
+        run: pytest --cov=atomate2 --cov-report=xml --ignore=tests/classical_md
+
+      - uses: codecov/codecov-action@v1
+        if: matrix.python-version == '3.10' && github.repository == 'materialsproject/atomate2'
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          file: ./coverage.xml
+
+  test_md:
```

Comment:
Many of the classical_md dependencies are not pip installable. We could consolidate the tests, but I'd need to modify the testing action to use micromamba instead of pip. I'll say more about this in another comment below.